charts/ingress-nginx/templates/controller-configmap.yaml

```diff
@@ -23,6 +23,9 @@ data:
 {{- if .Values.dhParam }}
   ssl-dh-param: {{ include "ingress-nginx.namespace" . }}/{{ include "ingress-nginx.controller.fullname" . }}
 {{- end }}
-{{- range $key, $value := .Values.controller.config }}
+{{- range $key, $value := merge .Values.controller.config .Values.configmap }}
```

Comment:
`configmap` has been deprecated a long time ago. We could forge a new major with this and PSP being removed, so this diff would also disappear.